_LowLevelCode/cpp/bin_pixels_c/bin_pixels_c.cpp

```diff
@@ -284,19 +284,23 @@ bool bin_pixels(double* s, double* e, double* npix,
             ok[i] = false;
             if (xt < cut_range[0] || xt >= cut_range[1]) {
                 if (xt == cut_range[1])xt *= (1. - FLT_EPSILON);
-                else continue;
+                else 
```

Comment:
I've modified code to put breakpoints on continue. To see what is rejected. I can revert it in hope nobody would ever need to do it in a future, but can we be sure?

On brackets I got used to Mantid convention, which does not use it on one-liners. Its indeed looks better, after getting use to it. 
Otherwise, the code is autoformatted by Visual studio.